ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/ant/tasks/main.yml

```diff
@@ -20,7 +20,6 @@
     mode: 0440
     timeout: 25
     validate_certs: no
-    checksum: sha512:2e48f9e429d67708f5690bc307232f08440d01ebe414059292b6543971da9c7cd259c21533b9163b4dd753321c17bd917adf8407d03245a0945fc30a4e633163
```

Comment:
Why remove this checksum?

Reply:
There was an error message that I thought it was causing - I checked yesterday and I was wrong so I'll put it back in :)

Reply:
Running again just to ensure the error isn't there: https://ci.adoptopenjdk.net/job/Will-QEMUplaybookTest/20/